components/server/src/bitbucket/bitbucket-repository-provider.ts

```diff
@@ -26,6 +26,11 @@ export class BitbucketRepositoryProvider implements RepositoryProvider {
         return { host, owner, name, cloneUrl, description, avatarUrl, webUrl };
     }
 
+    async getBranch(user: User, owner: string, repo: string, branch: string): Promise<Branch> {
+        // todo
+        throw new Error("not implemented");
```

Comment:
Q: When will we implement the Bitbucket and GitLab methods?

Reply:
Planing to implement GitLab support tomorrow. 

No plans for BitBucket so far. TODO: remove from selection list for the time being.